sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -203,6 +205,7 @@ public void processTimer(
     currentTimer = timer;
     currentTimeDomain = timeDomain;
     try {
+      @SuppressWarnings("unchecked")
```

Comment:
Please revert unrelated changes in this file.

Reply:
done